start.sh

```diff
@@ -0,0 +1,3 @@
+dune build || exit 1
```

Comment:
We can just use `dune exec src/bin/main.exe` instead, but this is not a blocker.